frontend/src/components/Thought.jsx

```diff
@@ -0,0 +1,34 @@
+import moment from "moment";
+import { useState } from "react";
+import { LikeButton } from "./LikeButton";
+
+export const Thought = ({ thought }) => {
```

Comment:
This fits very well into this kind of app, but please add a loader since Render is so slow.